README.md

```diff
@@ -86,19 +86,6 @@ New additions will be met with skepticism. If you think there is a
 missing feature, please file a bug clearly describing the problem and 
 the alternatives you tried before submitting a PR.
 
-# Reporting security issues
```

Comment:
Replaced with a stub referencing the CONTRIBUTING section with 8e7ab5b → caabd2a.  I think we *do* want the canonical docs in CONTRIBUTING though, since GitHub will show a link to those docs to anyone filing a new issue.